rpcq/_client.py

```diff
@@ -27,19 +27,20 @@
 
 _log = logging.getLogger(__name__)
 
+DEFAULT_TIMEOUT = 20_000  # ms
+
 
 class Client:
     """
     Client that executes methods on a remote server by sending JSON RPC requests to a socket.
     """
-    def __init__(self, endpoint: str, timeout: float = None):
```

Comment:
I don't think this should be removed or changed. I think whoever creates this Client should still have the chance to set whatever default timeout they would like, for whatever use case they have in mind. As Peter said before, this should remain as general as possible for multiple use cases. `DEFAULT_TIMEOUT` can also be taken out, or, the value can be passed as a default to the `timeout` parameter, to make it optional. If you wanted to be really careful, you could check that the given timeout isn't 0.

Reply:
I had thought @karalekas had suggested using this default instead of passing in a timeout. 

But fair point, I can undo...